org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,294 @@
+{
+    "app-id": "org.pitivi.Pitivi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.22",
```

Comment:
We do not want vaapi (not robust enough for us), our official development environment is flatpak and for a specific release we target a specific runtime/sdk version, changing randomly afterward is like what used to happen with distro, and this is really something I do not want.